src/current/cockroachcloud/resource-usage.md

```diff
@@ -39,6 +41,88 @@ To understand these resources, you need to understand a bit about the CockroachD
 
 ## Diagnose and optimize your resource consumption
 
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="standard">CockroachDB Standard</button>
```

Comment:
nit: Can we reorder the sections to have Basic first? To maintain consistency with the rest of the docs

Reply:
For this PR, I will keep the order of the sections: Standard, then Basic, because of [DOC-3381](https://cockroachlabs.atlassian.net/browse/DOC-3381) mentioned above. I created issue [DOC-9880](https://cockroachlabs.atlassian.net/browse/DOC-9880) to address this and other issues related to the Resource Usage page.